libnavui-speedlimit/src/main/res/layout/mapbox_speed_info_view.xml

```diff
@@ -44,12 +46,13 @@
                 android:layout_width="0dp"
                 android:layout_height="wrap_content"
                 android:gravity="center"
+                android:maxLines="1"
                 android:includeFontPadding="false"
                 app:layout_constraintEnd_toEndOf="parent"
                 app:layout_constraintStart_toStartOf="parent"
                 app:layout_constraintTop_toBottomOf="@id/postedSpeedLegend"
                 tools:textAppearance="@style/MapboxSpeedInfoPostedSpeedMutcdStyle"
```

Comment:
I see that the style defines text size as 32sp, while the width of the view is fixed and is specified in dp. Which means the speed value can easily not fit into the view if user increases text size in the system settings. That is how it looks for me (my current speed is actually 108, not 10):
<img src=https://user-images.githubusercontent.com/2395284/215534570-da2d8f84-410d-47b7-ad11-426fdfd4dd28.png width=360 />
If Drop-In doesn't support custom text sizes at the moment, then text size should be specified in dp.

Reply:
Good catch. Thx. Let me include that change as a part of this PR.

Reply:
Done

https://user-images.githubusercontent.com/2678039/215582268-90b10796-5e81-46ce-b13d-f3574cdbad5d.mp4